app/controllers/users/webauthn_setup_controller.rb

```diff
@@ -43,6 +43,7 @@ def new
       @need_to_set_up_additional_mfa = need_to_set_up_additional_mfa?
 
       if result.errors.present?
+        increment_mfa_selection_attempt_count(:webauthn)
```

Comment:
Should we also increment this in `confirm` below? In other controllers, we're incrementing on successful attempts. Right now we're only incrementing on frontend failures.